modules/channel-web/assets/default.css

```diff
@@ -233,6 +233,30 @@ body {
   align-items: flex-end;
 }
 
+.bpw-msg-list-loading {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  width: 50px;
+  height: 50px;
+  border: 3px solid rgba(0, 0, 0, 0.3);
+  border-radius: 50%;
+  border-top-color: rgb(0, 0, 0);
+  animation: spin 1s ease-in-out infinite;
+  -webkit-animation: spin 1s ease-in-out infinite;
+}
+
+@keyframes spin {
+  to {
+    -webkit-transform: rotate(360deg);
+  }
+}
+@-webkit-keyframes spin {
+  to {
+    -webkit-transform: rotate(360deg);
```

Comment:
Missing non-vendored prefixed transform rule
see [this guide](https://www.learnhowtoprogram.com/user-interfaces/movement-interactivity-frameworks/vendor-prefixes-for-animation)